src/main/kotlin/com/faforever/userservice/controller/OAuthController.kt

```diff
@@ -114,4 +119,14 @@ class OAuthController(
         }.flatMap { redirectUrl ->
             redirect(response, redirectUrl)
         }
+
+    @PostMapping("revokeSession")
+    @PreAuthorize("hasRole('" + FafRole.ADMIN_ACCOUNT_BAN + "') + hasRole('" + OAuthScope.ADMINISTRATIVE_ACTION + "')")
+    fun revokeSession(
```

Comment:
Sessions might be a misleading term in all cases. revokeConsent would be correct from the hydra perspective or revokeRefreshToken from client perspective.

Reply:
makes sense